x-pack/packages/ml/aiops_utils/src/get_window_parameters.ts

```diff
@@ -65,3 +65,46 @@ export const getWindowParameters = (
     deviationMax: Math.round(deviationMax),
   };
 };
+
+export const getSnappedWindowParameters = (
+  d: WindowParameters,
```

Comment:
```suggestion
  windowParameters: WindowParameters,
```

Reply:
Updated to clearer naming in 3f2a5c5034db76c0767c0c7cf5ee570214be066c